os/sdk-modifying-coreos.md

```diff
@@ -151,10 +151,10 @@ Build all of the target binary packages:
 
 #### Render the CoreOS Container Linux image
 
-Build an image based on the binary packages built above, including development tools:
+Build the production image based on the binary packages built above:
```

Comment:
nit: `a production image` rather than `the production image`.

Reply:
Got it ! :smile: